chain/network/src/routing/routing_table_view.rs

```diff
@@ -176,30 +176,28 @@ impl RoutingTableView {
     }
 
     pub(crate) fn add_ping(&mut self, ping: Ping) {
-        let cnt = self.ping_info.get(&(ping.nonce as usize)).map(|v| v.1).unwrap_or(0);
-
-        self.ping_info.put(ping.nonce as usize, (ping, cnt + 1));
+        if self.ping_info.len() < PING_PONG_CACHE_SIZE {
+            self.ping_info.push(ping.clone());
+        }
```

Comment:
This feels a bit weird as we never actually `pop` from the info, so pings become no-op starting from `PING_PONG_CACHE_SIZE`.

If this is reachable from prod, this feels like a surprising semantics. If this is for tests only, than maybe just assert here?

Reply:
@matklad PTAL